db/migrate/20190907020057_ensure_no_warpable_corruption.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '20111005211631_add_zip_warpable_res_history'
+
+class EnsureNoWarpableCorruption < ActiveRecord::Migration[5.2]
+  def self.up
+    AddZipWarpableResHistory.down
```

Comment:
Won't this flush the `zip` field in the `exports` table? And similarly for other migrations here? 

```ruby
    remove_column :warpables, :history	    remove_column :warpables, :history
    remove_column :warpables, :cm_per_pixel	    remove_column :warpables, :cm_per_pixel
    remove_column :exports, :zip	    remove_column :exports, :zip
```